modular_citadel/code/modules/reagents/chemistry/reagents/fermi_reagents.dm

```diff
@@ -340,10 +340,10 @@
 /datum/reagent/fermi/acidic_buffer
 	name = "Acidic buffer"
 	id = "acidic_buffer"
-	description = "This reagent will consume itself and move the pH of a beaker towards 3 when added to another."
+	description = "This reagent will consume itself and move the pH of a beaker towards acidity when added to another."
 	taste_description = "an acidy sort of taste, blech."
```

Comment:
This is going to sound redunant as "taste" is already being used as a verb at the start of to_chat output for the ingestion reaction.

Reply:
It's a toggle between the two spans, Depending on prefs.